src/pcre2_ucd.c

```diff
@@ -142,14 +142,22 @@ const uint32_t PRIV(ucd_caseless_sets)[] = {
   0x004b,  0x006b,  0x212a,  NOTACHAR,
   0x00c5,  0x00e5,  0x212b,  NOTACHAR,
   0x1c88,  0xa64a,  0xa64b,  NOTACHAR,
+  0x0069,  0x0130,  NOTACHAR,
+  0x0049,  0x0131,  NOTACHAR,
 };
 
+/* These are the indices, within ucd_caseless_sets, of the additional
+Turkish case-equivalences. */
+
+const uint32_t PRIV(ucd_turkish_dotted_i_caseset) = 112;
+const uint32_t PRIV(ucd_turkish_dotless_i_caseset) = 115;
```

Comment:
Is it worth to have two variables for this? A +3 could be enough, since they are together.

Reply:
Carlo would probably complain if he saw a hardcoded +3 :)

Reply:
You can use a `#define`

Reply:
Ironically, I think it is less ugly if hardcoded all the way like this, but sure hope we can get `OP_2CHAR` done soon enough to remove this ASAP